### PR TITLE
Edit profile assessments admin

### PR DESCRIPTION
**Pull request by Blackfry**
Highlight if changes were made to seller profile case studies

### CODE REVIEW

File: src/bundles/SellerRegistration/components/ApplicationPreview/ApplicationPreview.js [hunk 1]
```diff
@@ -23,6 +54,9 @@ const ApplicationPreview = ({ header, body, privateInfo, onClick }) => (
         <Link to="/review">Go back and edit</Link>
       </div>
     </div>}
+    <div>
+      <Changes {...body}/>
```
Comment (maxious) [inline src/bundles/SellerRegistration/components/ApplicationPreview/ApplicationPreview.js]: Does this show up in the public view of the profile? You could use the same logic as PrivateInfo to hide {!onClick && privateInfo.disclosures

Comment (Blackfry) [inline src/bundles/SellerRegistration/components/ApplicationPreview/ApplicationPreview.js]: it renders conditionally on unassessed case studies which public profile shouldn't have but I'll take a look at that

**Comment by matt-sm** [comment]
there are more changes after discussion with Neil?

**Comment by Blackfry** [comment]
This was going to v1 with v2 pulling out details of changes made via json diffing. Depends if they are happy to wait for that

**Pull request merged**